message_tools/find_invalid_address_case_id_by_uprn.py

```diff
@@ -0,0 +1,73 @@
+import argparse
```

Comment:
This shouldn't live in message_tools, bulk_processing seems like the suitable place.